libraries/AP_BattMonitor/AP_BattMonitor.cpp

```diff
@@ -330,6 +330,27 @@ float AP_BattMonitor::voltage(uint8_t instance) const
     }
 }
 
+/// voltage - returns average cell battery voltage in volts
+float AP_BattMonitor::cell_avg_voltage(uint8_t instance) const
+{
+    if ((instance < _num_instances) && (state[instance].cell_count)) {
```

Comment:
Do an early-return based on these things being out-of-range rather than the `else` hidden right down the bottom

Reply:
Ok, I changed that and I also modified voltage() to early return. The only reason I did it that way originally was because voltage() was written that way.